src/components/ErrorForm/ErrorFormInlineError.jsx

```diff
@@ -1,77 +1,59 @@
-import React from 'react'
+import React, { useRef, useState } from 'react'
 import formStyle from './errorForm.module.css'
 
-class ErrorFormInlineError extends React.Component {
-  emailRef = React.createRef()
+export default function ErrorFormInlineError() {
+  const emailRef = useRef(null)
+  const [emailInputValid, setEmailInputValid] = useState(false)
```

Comment:
I think this needs to be undefined, to stop the error state being shown by default (I put some more details on the other file below about how you can see this issue in the deploy preview) :)